person.rb

```diff
@@ -20,6 +21,10 @@ def correct_name
     name
   end
 
+  def add_rental(rental)
+    rentals << rental
```

Comment:
- [ ] The `rentals` here should be the instance variable.

  ```
  @rentals << rental
  ```

Reply:
done